build-workspace.js

```diff
@@ -11,16 +11,16 @@ const executeScript = require('./lib/executeScript');
 const requireJSONIn = require('./lib/requireJSON');
 
 const tools = {
-  executeScript: executeScript
-, repoHash: repoHash
+  executeScript: executeScript,
+  repoHash: repoHash
 };
 
 const buildWorkspace = (
```

Comment:
![image](https://user-images.githubusercontent.com/5292838/55712663-71094700-59ef-11e9-976b-5ef8c2b1a910.png)
This function is imo way too complex ;)